src/cpp/session/modules/NotebookPlots.R

```diff
@@ -29,7 +29,14 @@
       units    = units,
       res      = 96 * pixelRatio
    )
-   
+
+   gdBackend <- getOption("RStudioGD.backend")
+   if (gdBackend != "default")
```

Comment:
```suggestion
   if (!identical(gdBackend, "default"))
```

Just to guard against cases where `gdBackend` might not be a string. (this shouldn't happen but better safe than sorry)